lib/membrane_opus/decoder.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Membrane.Opus.Decoder do
+  @moduledoc """
+  This element performs decoding of Opus audio.
+  """
+
+  use Membrane.Filter
+
+  alias __MODULE__.Native
+  alias Membrane.Buffer
+  alias Membrane.Caps.Audio.Raw
+
+  @avg_opus_packet_size 960
```

Comment:
Just out of curiosity - does this come from our beloved Insitiute (you should know which one)? Or was it measured? Maybe we could measure it somehow in runtime?

Reply:
This is sponsored by the Institute :D Once we have automated demand calculation this will be done by the core